src/qml/TrackerSettings.qml

```diff
@@ -21,6 +21,31 @@ Popup {
   closePolicy: Popup.CloseOnEscape
 
   property var tracker: undefined
+
+  Connections {
+    target: trackingModel
+
+    function onTrackingSettingsRequested(trackerIndex, skipSettings) {
+      tracker = trackings.itemAt(trackerIndex.row).tracker
+      if (!skipSettings) {
+        trackerSettings.open()
+        trackerSettings.focus = true
+      } else {
+        featureModel.resetAttributes()
+        tracker.feature = featureModel.feature
+        if (embeddedAttributeFormModel.rowCount() > 0 && !featureModel.suppressFeatureForm()) {
+          embeddedFeatureForm.active = true
+        } else {
+          trackingModel.startTracker(tracker.vectorLayer)
+          displayToast(qsTr('Track on layer %1 started').arg(tracker.vectorLayer.name))
+          if (featureModel.currentLayer.geometryType === Qgis.GeometryType.Point) {
+            projectInfo.saveTracker(featureModel.currentLayer)
+          }
+        }
+      }
+    }
+  }
```

Comment:
```suggestion
    function onTrackingSettingsRequested(trackerIndex, skipSettings) {
      tracker = trackings.itemAt(trackerIndex.row).tracker

      if (!skipSettings) {
        trackerSettings.open()
        trackerSettings.focus = true
      } else {
        featureModel.resetAttributes()
        tracker.feature = featureModel.feature

        if (embeddedAttributeFormModel.rowCount() > 0 && !featureModel.suppressFeatureForm()) {
          embeddedFeatureForm.active = true
        } else {
          trackingModel.startTracker(tracker.vectorLayer)

          displayToast(qsTr('Track on layer %1 started').arg(tracker.vectorLayer.name))

          if (featureModel.currentLayer.geometryType === Qgis.GeometryType.Point) {
            projectInfo.saveTracker(featureModel.currentLayer)
          }
        }
      }
    }
  }
```

Matter of taste, I find having  a bit of air being more readable, but no strong opinion.